data/trusted-types.json

```diff
@@ -0,0 +1,6 @@
+{
+    "url": "https://wicg.github.io/trusted-types/dist/spec/",
```

Comment:
The `title` property is needed because the framework does not know how to retrieve the title of WICG specs.